zoobot/pytorch/training/train_with_pytorch_lightning.py

```diff
@@ -13,7 +13,8 @@
 from zoobot.pytorch.datasets import decals_dr8
 from zoobot.pytorch.training import losses
 from zoobot.pytorch.estimators import define_model
-from zoobot.pytorch.estimators import resnet_detectron2_custom, efficientnet_standard, resnet_torchvision_custom
+# from zoobot.pytorch.estimators import resnet_detectron2_custom, efficientnet_standard, resnet_torchvision_custom
```

Comment:
I removed this only for the unused dependency install...once that is back in i can re import the `resnet_detectron2_custom` properly.